x-pack/plugins/triggers_actions_ui/public/plugin.ts

```diff
@@ -193,6 +195,9 @@ export class Plugin
 
     registerBuiltInActionTypes({
       actionTypeRegistry: this.actionTypeRegistry,
+      services: {
+        validateEmailAddresses: plugins.actions.validateEmailAddresses,
```

Comment:
Since this validation is used on the client side, it seems like it should also allow mustache templates to bypass validation, right?

```suggestion
        validateEmailAddresses: (addresses) =>
          plugins.actions.validateEmailAddresses(addresses, {
            dangerouslyAllowMustacheBypass: true,
          }),
```

Reply:
Only problem with this is if we add more options, we may not want to default them all right here.  But we only have this option now.  Probably worth doing your way, we can figure out a safer approach if we need to pass more options later.

Reply:
Actually, this made me realize we weren't validating `from` (not allowed to use mustache since it's a config property, not a param), so I had to make a few other changes.